kubernetes-kubectl.el

```diff
@@ -276,6 +293,23 @@ ERROR-CB is called if an error occurred."
                           (funcall cb (match-string 1 (buffer-string)))))
                       error-cb))
 
+
+(defun kubernetes-kubectl-delete-ingress (props state ingress-name cb &optional error-cb)
+  "Delete INGRESS-NAME, then execute CB with the response buffer.
+
+PROPS is an alist of functions to inject.  It should normally be passed
+`kubernetes-props'.
+
+STATE is the application state.
+
+ERROR-CB is called if an error occurred."
+  (kubernetes-kubectl props state (list "delete" "ingress" secret-name "-o" "name")
```

Comment:
I think copy paste typo here. This should be ingress-name.

Reply:
Thanks Updated it